jSka/ingest/archive.py

```diff
@@ -0,0 +1,170 @@
+import os
+import numpy as np
+
+from pathlib import Path
+
+import tables
+from tables import *
+import tables3_api
+
+import pyyaks.logger
+
+
+loglevel = pyyaks.logger.VERBOSE
+logger = pyyaks.logger.get_logger(name='jskaarcive', level=loglevel,
+                                  format="%(asctime)s %(message)s")
+
+class DataProduct:
```

Comment:
Wondering if this need to be a class since it all just static methods.  I saw this talk at PyCon years ago and it made an impression that stuck:
https://www.youtube.com/watch?v=o9pEzgHorH0

The *module-level* compartmentalization of these functions could be sufficient.

Reply:
I think I agree, and will refactor. I was thinking that different output DataProducts would be supported, but at this point I don't think that is the case.